app/views/time_regs/index.html.erb

```diff
@@ -47,12 +47,19 @@
         <div class="flex lg:text-3xl text-xl w-3/4  max-[800px]:hidden w-full">
           <% if @chosen_date == Date.today %>
             <p class="font-medium mr-4">
-            Today: 
+            Today:
             </p>
+            <p class="  ">
+              <%= @chosen_date.strftime("%A, %d %B") %>
+            </p>
+          <% else %>
+            <%= @chosen_date.strftime("%A, %d %B") %>
+          <button data-controller="date-redirect" data-action="click->date-redirect#redirectToToday">
+            <div class="ml-2 text-blue-600 ">
+              <%= submit_tag "Go to today", class: "text-lg underline underline-offset-2 cursor-pointer" %>
+            </div>
+          </button>
```

Comment:
Is it possible to just use a normal link instead?

Like this:
`<%= link_to "Go to today", root_path %>`